port/protobuf.h

```diff
@@ -20,16 +20,19 @@
 #include "google/protobuf/any.pb.h"
 #include "google/protobuf/descriptor.pb.h"
 #include "google/protobuf/message.h"
+#include "google/protobuf/stubs/common.h"  // for GOOGLE_PROTOBUF_VERSION
 #include "google/protobuf/text_format.h"
 #include "google/protobuf/util/message_differencer.h"
 #include "google/protobuf/wire_format.h"
 
+#if GOOGLE_PROTOBUF_VERSION >= 3007000  // commit 6bbe197e9c1b6fc38cbdc45e3bf83fa7ced792a3 of >=3.7.0
 // clang-format off
-#include "google/protobuf/port_def.inc"  // MUST be last header included
+# include "google/protobuf/port_def.inc"  // MUST be last header included
```

Comment:
I though we need this for PROTOBUF_VERSION
if we switch to GOOGLE_PROTOBUF_VERSION, can we drop this include?

Reply:
@vitalybuka I don't know. I can drop it, throw it at the CI and see if it sticks. Protobuf examples want us to add it though, so we should probably keep it?

Reply:
@vitalybuka PS: I confirm that offering `PROTOBUF_VERSION` is one of the effects of that include, yes.

Reply:
what kind of examples?

Reply:
@vitalybuka seems like I mis-remembered the headers own usage instructions at https://github.com/protocolbuffers/protobuf/blob/7229ef1b4f0f975036b140fe71c76d25feff4918/src/google/protobuf/port_def.inc#L19-L23 as a tutorial. I cannot find other places promoting it. Let me query CI for a version without it…

Reply:
@vitalybuka CI started at https://github.com/hartwork/libprotobuf-mutator-fork/actions/runs/13424660914 …

Reply:
> I though we need this for PROTOBUF_VERSION if we switch to GOOGLE_PROTOBUF_VERSION, can we drop this include?

@vitalybuka dropped and pushed now